### PR TITLE
kast: prettifications

### PR DESCRIPTION
**Pull request by asymmetric**
* Adds `maxUInt256 - 255`, used when zeroing one-byte-long values.
* Prettifies `WordPack`

### CODE REVIEW

File: lib/kast.js [hunk 1]
```diff
@@ -282,6 +283,8 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       "intList" : "",
       ".List{\"intList\"}": "",
       ".WordStack_EVM-DATA": "",
+      "#WordPackUInt112UInt112UInt32(_,_,_)_RULES": "#WordPackUInt112UInt112UInt32",
```
Comment (xwvvvvwx) [inline lib/kast.js]: not sure we should be putting application specific rules into `klab` tbh

File: lib/kast.js [hunk 2]
```diff
@@ -246,6 +246,7 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
     const pt = {
       "115792089237316195423570985008687907853269984665640564039457584007913129639936": "pow256",
       "115792089237316195423570985008687907853269984665640564039457584007913129639935": "maxUInt256",
+      "115792089237316195423570985008687907853269984665640564039457584007913129639680": "maxUInt256 - 255",
```
Comment (xwvvvvwx) [inline lib/kast.js]: I'm not sure if this prettification isn't too specific?

Comment (asymmetric) [reply lib/kast.js]: I think this one is not overly specific.

File: lib/kast.js (hunk 1 above)
Comment (asymmetric) [reply lib/kast.js]: I can agree about this one.

File: lib/kast.js [hunk 3]
```diff
@@ -282,6 +283,8 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       "intList" : "",
       ".List{\"intList\"}": "",
       ".WordStack_EVM-DATA": "",
+      "#WordPackUInt112UInt112UInt32(_,_,_)_RULES": "#WordPackUInt112UInt112UInt32",
+      "_[_:=_]_EVM-DATA": "EVM-DATA",
```
Comment (asymmetric) [inline lib/kast.js]: I'm not sure this is a good idea anymore. `EVM-DATA` is the module name, it make no sense to remove the rest and leave this in.

Maybe this should be prettified as `[ := ]`?

File: lib/kast.js (hunk 2 above)
Comment (asymmetric) [reply lib/kast.js]: We can leave this out if we think people will figure out quickly enough that this is `maxUInt256 - 255`. It took me a few minutes.

Is your concern scalability, i.e. that we'll have to add a whole bunch of `maxUInt256 - X` numbers? That's possible.

File: lib/kast.js [hunk 4]
```diff
@@ -282,6 +283,8 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       "intList" : "",
       ".List{\"intList\"}": "",
       ".WordStack_EVM-DATA": "",
+      "#WordPackUInt112UInt112UInt32(_,_,_)_RULES": "#WordPackUInt112UInt112UInt32",
+      "_[_:=_]_EVM-DATA": "[ := ]",
```
Comment (asymmetric) [inline lib/kast.js]: Not super happy about this, suggestions welcome.

File: lib/kast.js (hunk 1 above)
Comment (desaperados) [reply lib/kast.js]: I think I prefer to see the K version in this case. I think we should be careful not to hide meaning.

File: lib/kast.js (hunk 2 above)
Comment (desaperados) [reply lib/kast.js]: Agree that this improves readability.

Comment (xwvvvvwx) [reply lib/kast.js]: > Is your concern scalability, i.e. that we'll have to add a whole bunch of maxUInt256 - X numbers? That's possible.

This is my concern yes. I'm happy to merge and then revisit if it starts becoming an issue.

> Could it be confused for a stack operation?

This is a valid point IMO.

File: lib/kast.js (hunk 4 above)
Comment (MrChico) [reply lib/kast.js]: should be infix notated

**Comment by asymmetric** [comment]
Closing, these turned out not to be super useful.

**Pull request closed without merging**